Simulations/EUREC4A-MIP/warming/setup.yaml

```diff
@@ -0,0 +1,22 @@
+plugins:
+  source:
+    - module: intake_xarray
+
+sources:
+
+  boundary_conditions:
+    args:
+      urlpath: simplecache::swift://swift.dkrz.de/dkrz_0913c8f3-e7b6-4f94-9221-06880d4ccfea/PGW+4K/cas{{date.strftime("%Y%m%d%H%M%S")}}.nc
+      chunks: null
+      xarray_kwargs:
+        engine: h5netcdf
+    parameters:
+      date:
+        description: date
+        default: 2020-01-01
+        type: datetime
+        min: 2020-01-01
+        max: 2020-03-01
+    description: Boundary conditions of 4K warming experiments
```

Comment:
@torresalavez, do I understand correctly that these PGW+4K fields can be used as is and are not just tendencies that need to be added to the boundary conditions from the control setup, right?

Reply:
@torresalavez my assumption here is correct, right?

Reply:
Based on the values in the datasets I assume my assumption is correct.